### ISSUE TITLE
Android : D_Pad_Center key is missing

### ISSUE THREAD

**Issue by OscarLeif** [cocos/cocos-engine#16505]
### Cocos Creator version

3.8.1

### System information

Android.

### Issue description

When create a navigation UI for Cocos.
Using a Gamepad works fine.
This is required for Android TV Platform.

KEYCODE_DPAD_CENTER = missing.
KEYCODE_DPAD_DOWN = OK
KEYCODE_DPAD_LEFT = OK
KEYCODE_DPAD_RIGHT = OK
KEYCODE_DPAD_UP = OK

In the next example we have that we can use the Android Gamepad on the Android Platform.

` if(event.gamepad.dpad.center.getValue() == 1)//missing Android Gamepad_input`

```
initListener (): void
    {     
        input.on(Input.EventType.KEY_DOWN, this.OnKeyDown, this);
        input.on(Input.EventType.TOUCH_START, this.touchStart, this);      
        input.on(Input.EventType.GAMEPAD_INPUT, this.GamePadOnKeyDown, this);    
    }

    private GamePadOnKeyDown (event: EventGamepad): void
    {
        if (event.gamepad.buttonSouth.getValue() == 1 )
        {
            this.FlapBird();
        }
      //The DpadCenter is not mapped.
      if(event.gamepad.dpad.center.getValue() == 1)

      //But this values are mapped
     let leftAction = event.gamepad.dpad.left.getValue() == 1;
     let rightAction = event.gamepad.dpad.right.getValue() == 1;
     let upAction = event.gamepad.dpad.up.getValue() == 1;
     let downAction = event.gamepad.dpad.down.getValue() == 1;

    }
```

### Steps to reproduce

Basically this is just On Android TV platform.
We cannot use the remote to submit an action using the center button on most of remote devices.
